### PR TITLE
Add DEA e2e

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Add e2e tests for "Which GI Bill benefit do you want to use?" selection "DEA" (chapter 35) in `src/applications/gi/tests/e2e/04-dea.e2e.spec.js`

This test checks housing rates for an OJT institution and a PUBLIC institution when inputs are changed.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/tests/e2e/chapter33-helpers.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const Timeouts = require('../../../../platform/testing/e2e/timeouts');
+const E2eHelpers = require('../../../../platform/testing/e2e/helpers');
```
Comment (shawkey-daniel) [inline src/applications/gi/tests/e2e/chapter33-helpers.js:2]: Please use the platform alias here.

File: src/applications/gi/tests/helpers.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import createCommonStore from '../../../platform/startup/store';
```
Comment (shawkey-daniel) [inline src/applications/gi/tests/helpers.js:1]: Please use the platform alias here.

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/gi/tests/e2e/chapter33-helpers.js (hunk 1 above)
Comment (zurbergram) [reply src/applications/gi/tests/e2e/chapter33-helpers.js]: changed

File: src/applications/gi/tests/helpers.js (hunk 2 above)
Comment (zurbergram) [reply src/applications/gi/tests/helpers.js]: changed

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Pull request merged**